tests/integration/security/fuzz/fuzzers/dotdotpwn/dotdotpwn.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dotdotpwn
+spec:
+  selector:
+    matchLabels:
+      app: dotdotpwn
+  template:
+    metadata:
+      labels:
+        app: dotdotpwn
+      annotations:
+        "sidecar.istio.io/inject": "false"
+    spec:
+      containers:
+        - name: dotdotpwn
+          # TODO: Upload the image to istio-ecosystem.
+          image: gcr.io/ymzhu-istio/dotdotpwn:latest
```

Comment:
Here, it's listed as TODO, but I'd think this should not be going to a personal repo when it's merged.

Reply:
Yes, we should definitely not use the personal repo. 

I'm thinking could we leave it as-is for now because I need to add a few more images soon and I prefer to get these images uploaded all at once when ready?

Reply:
updated to use gcr.io/istio-testing/dotdotpwn